apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -405,18 +511,18 @@ int smartfs_powercut_main(int argc, char *argv[])
 		/* Small test use same as size of data area of mtd sector, so there should not be error
 		 * during the verify for small file test because it protected by journal.
 		 */
-		ret = do_test(TEST_SMALL, TEST_SMALL_BACKUP, TEST_SMALL_BACKUP_DIR, TEST_SMALL_BUFSIZE);
+		ret = do_test(TEST_SMALL, TEST_SMALL_BACKUP, TEST_SMALL_BACKUP_DIR, TEST_SMALL_BUFSIZE, FILE_TYPE_RANDOM);
 		if (ret != OK) {
 			printf("test small file failed\n");
-			break;
+			return ret;
 		}
 
-		ret = do_test(TEST_LARGE, TEST_LARGE_BACKUP, TEST_LARGE_BACKUP_DIR, TEST_LARGE_BUFSIZE);
+		ret = do_test(TEST_LARGE, TEST_LARGE_BACKUP, TEST_LARGE_BACKUP_DIR, TEST_LARGE_BUFSIZE, FILE_TYPE_RANDOM);
 		if (ret != OK) {
 			printf("test large file failed\n");
-			break;
+			return ret;
 		}
 	}
 	
-	return ret;
+	return OK;
```

Comment:
Line accessible?

Reply:
Can't understand what you mean exactly, but if all files and directories created properly, or already exist, it returns ok.

Reply:
I meant is this line of code reachable?
Since the while loop now only exits when one of the tests returns an error

Reply:
function type is int, it should return something or warning or build error will happen.